gpustack/policies/candidate_selectors/vllm_resource_fit_selector.py

```diff
@@ -286,6 +242,73 @@ def _set_messages(self):
                 "Cannot find a suitable worker combination to run the model in distributed mode. If you are confident that the resources are sufficient, you may manually schedule the model by selecting the workers and GPUs."
             )
 
+    def _build_non_llm_message(self) -> str:
+        # Non-LLM models
+        default_message = f"- The model requires approximately {byte_to_gib(self._vram_claim)} GiB of VRAM.\n"
+        if self._largest_multi_gpu_vram > 0:
+            default_message += f"- The largest available worker has {byte_to_gib(self._largest_multi_gpu_vram)} GiB of VRAM.\n"
+        elif self._largest_single_gpu_vram > 0:
+            default_message += f"- The available GPU has {byte_to_gib(self._largest_single_gpu_vram)} GiB of VRAM.\n"
+        return default_message
+
+    def _cal_effective_vram(self) -> float:
+        if self._largest_multi_gpu_total == 0:
+            return 0.0
+        return (
+            byte_to_gib(self._largest_multi_gpu_vram)
+            * self._gpu_memory_utilization
+            * self._largest_multi_gpu_utilization_satisfied_count
+            / self._largest_multi_gpu_total
+        )
+
+    def _build_gpu_message(self):
```

Comment:
Can we add unit tests in test_vllm_resource_fit_selector.py to check the messages?